pkg/api/v1/types.go

```diff
@@ -1258,6 +1258,19 @@ type PodSpec struct {
 
 // PodSecurityContext holds pod-level security attributes and common container settings.
 type PodSecurityContext struct {
+	// SupplementalGroups can be used to specify a list of
+	// additional groups which the main container process will run
```

Comment:
... additional groups which are applied to the first-run process of each user container in the pod. This is in addition to the primary group of each container.